insights/static/js/components/bar-chart.js

```diff
@@ -41,6 +41,9 @@ export const barChart = {
                             display: false,
                         },
                         ticks: {
+                          userCallback(value) {
+                            return value.toLocaleString()
```

Comment:
Is this so that the page isn't rendered and then a second update happens to run toLocaleString?

Reply:
It applies accountancy commas to the Y axis labels, which can't be accessed in the same way as the other values.